web/src/main/resources/openmrs-servlet.xml

```diff
@@ -91,7 +91,9 @@
  	<bean class="org.springframework.web.servlet.handler.SimpleMappingExceptionResolver"> 
  		<property name="exceptionMappings"> 
  			<map> 
- 				<entry key="org.openmrs.api.APIException" value="uncaughtException"/> 
+ 				<entry key="org.openmrs.api.APIException" value="uncaughtException"/>
+				<entry key="org.springframework.core.NestedRuntimeException" value="uncaughtException"/>
```

Comment:
What is the value of having both `org.springframework.core.NestedRuntimeException` and `java.lang.Exception`?

Reply:
@dkayiwa resolved